src/components/radios/_macro.spec.js

```diff
@@ -122,6 +122,16 @@ const EXAMPLE_RADIO_ITEM_RADIOS = {
     },
 };
 
+const EXAMPLE_RADIO_ITEM_WITH_PRESET_TEXTAREA = {
+    other: {
+        otherType: 'textarea',
+        checked: true,
+        id: 'other-textbox-radio-with-text-input',
```

Comment:
```suggestion
        id: 'other-textarea-input',
```
Should be named something like this to be consistent with other examples and it specifies its a textarea not a text input